aiida/tools/importexport/dbimport/backends/django.py

```diff
@@ -38,7 +38,7 @@
 
 def import_data_dj(
     in_path: str,
-    group: Optional[Group] = None,
+    group: Optional[ImportGroup] = None,
```

Comment:
I reverted this change as it is backwards incompatible. We are currently allowing users to use a plain `Group` to store nodes in upon importing. We cannot narrow this restriction down.